scripts/apps/authoring/authoring/modified-info.tsx

```diff
@@ -11,7 +11,7 @@ export class ModifiedInfo extends React.PureComponent<IProps> {
     render() {
         const {article} = this.props;
 
-        if (article.versioncreated == null) {
+        if (article.versioncreated == null || article.pubstatus === 'CANCELED') {
```

Comment:
What is the reason to now show the versioncreated if the item has been killed?

Reply:
mistake, forgot to commit the fix